subprojects/model-groovy/src/integTest/groovy/org/gradle/cli/TaskOptionsSpec.groovy

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.cli
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+
+class TaskOptionsSpec extends AbstractIntegrationSpec {
+    def "can use a built-in option"() {
+        when:
+        buildScript """
+            abstract class MyTask extends DefaultTask {
```

Comment:
You should switch this task definition to use the more idiomatic def in the next test.  Extract it to a `defineTaskWithProfileOption()` private method and just call that from all tests using it.  Then `when` clause here then becomes:

```groovy
when:
buildScript """${defineTaskWithProfileOption())}

tasks.register('mytask', MyTask.class)
```

Reply:
Brilliant. I was trying to figure out if there was a way to write this in a single buildscript and include it in multiple places... easier to just include the string!